resin_calculator/artifact_calc.py

```diff
@@ -26,23 +29,24 @@ def __init__(self, rarity, artifact_set, artifact_slot, main_stat, off_stats):
 
 def user_input(message):
     user_response = input(message)
-    user_response = user_response.lower()
+    user_response = user_response.lower() #converts all inputs to lowercase to error-proof user inputs
     return user_response
 
 def greeting():
     print("Welcome to the Artifact Calculator.\n")
 
 def select_artifact_slot():
     valid_slot = False
-    slots = ["circlet", "timepiece", "goblet", "feather", "flower"]
+    valid_artifact_slots = ["circlet", "timepiece", "goblet", "feather", "flower"]
```

Comment:
Same here, I would put this with the rest of the built-in data.

Reply:
Relocated!